tests/acceptance/bootstrap/CliContext.php

```diff
@@ -233,4 +234,88 @@ public function theCommandOutputShouldContain(string $shouldOrNot, string $outpu
 			Assert::assertStringNotContainsString($output, $jsonResponse["message"]);
 		}
 	}
+
+	/**
+	 * @When the administrator lists all the upload sessions
+	 * @When the administrator lists all the upload sessions with flag :flag
+	 *
+	 * @param string|null $flag
+	 *
+	 * @return void
+	 */
+	public function theAdministratorListsAllTheUploadSessions(?string $flag = null): void {
+		if ($flag) {
+			$flag = "--$flag";
+		}
+		$command = "storage-users uploads sessions --json $flag";
+		$body = [
+			"command" => $command
+		];
+		$this->featureContext->setResponse(CliHelper::runCommand($body));
+	}
+
+	/**
+	 * @When the administrator cleans all expired upload sessions
+	 *
+	 * @return void
+	 */
+	public function theAdministratorCleansAllExpiredUploadSessions(): void {
+		$command = "storage-users uploads sessions --expired --clean --json";
+		$body = [
+			"command" => $command
+		];
+		$this->featureContext->setResponse(CliHelper::runCommand($body));
+	}
+
+	/**
+	 * @Then /^the CLI response should (not|)\s?contain these entries:$/
```

Comment:
should be more readable
```suggestion
	 * @Then /^the CLI response (should|should not) contain these entries:$/
```